Userland/Libraries/LibDSP/Track.cpp

```diff
@@ -64,6 +64,10 @@ bool NoteTrack::check_processor_chain_valid() const
 ErrorOr<void> Track::resize_internal_buffers_to(size_t buffer_size)
 {
     m_secondary_sample_buffer = TRY(FixedArray<Sample>::create(buffer_size));
+    bool false_variable = false;
+    while (!m_sample_lock.compare_exchange_strong(false_variable, true)) { }
```

Comment:
This loop should contain a short sleep of e.g. half a microsecond, see `usleep`. Otherwise, this unnecessarily uses up processor time.